assets/src/scss/layout/_breadcrumbs.scss

```diff
@@ -20,19 +20,17 @@
     position: relative;
   }
 
-  .tag-item--main + .tag-wrap,
-  .tag-wrap + .tag-wrap {
-    &:before {
-      content: "\2022";
-      pointer-events: none;
-      display: inline-block;
-      margin-left: -12px;
-      margin-right: 8px;
+  .tag-wrap-bullet {
+    pointer-events: none;
+    display: inline-block;
+    color: $link-color;
```

Comment:
We didn't use link color for bullets before, no need for it here

Reply:
Thank you for catching this, still getting used to testing these changes! ❤️